pkiinit/pkiInit_test.go

```diff
@@ -67,6 +135,19 @@ func (testExit *testExitCode) getStatusCode() int {
 	return testExit.testStatusCode
 }
 
-func newTestExit(exit exit) exit {
-	return &testExitCode{}
+type testPkiInitOptionDispatcher struct {
+	pkiInitOptionDispatcher
+}
+
+func newTestDispatcher(dispatcher optionDispatcher) optionDispatcher {
+	return &testPkiInitOptionDispatcher{}
```

Comment:
parameters could be removed. see comment for newOptionDispatcher() in pkiInit.go